modules/io/adaptors/parse_dataframe_to_bytes.py

```diff
@@ -58,11 +73,12 @@ def parse_dataframe(vineyard_socket, stream_id, proc_num, proc_index):
 
 if __name__ == '__main__':
     if len(sys.argv) < 5:
-        print('usage: ./parse_dataframe_to_bytes <ipc_socket> <stream_id> <proc_num> <proc_index>')
+        print('usage: ./parse_dataframe_to_bytes <ipc_socket> <stream_id> <file_path> <proc_num> <proc_index>')
```

Comment:
I think the "parse_dataframe_to_bytes" should take a stream and yeild a stream? Why it requires a `file_path`?